File: bakalarkaBE/bakalarkaBE/Models/Odporucacilistok.cs
using System;
using System.Collections.Generic;

namespace bakalarkaBE.Models;

public partial class Odporucacilistok
{
    public DateOnly Datumodporucenia { get; set; }

    public string Kododdelenia { get; set; } = null!;

    public string Idnemocnice { get; set; } = null!;

    public string Osobnecislo { get; set; } = null!;

    public string Rodnecislo { get; set; } = null!;

    public virtual Oddelenie Oddelenie { get; set; } = null!;

    public virtual Doktor OsobnecisloNavigation { get; set; } = null!;

    public virtual Pacient RodnecisloNavigation { get; set; } = null!;
}

File: FE/src/app/models/OdporucaciListok.ts
import {Oddelenie} from "./Oddelenie";
import {Doktor} from "./Doktor";
import {Pacient} from "./Pacient";

export class OdporucaciListok {
  datumodporucenia!: Date;
  kododdelenia: string = "";
  idnemocnice: string = "";
  osobnecislo: string = "";
  rodnecislo: string = "";
  oddelenie?: Oddelenie;
  osobnecisloNavigation?: Doktor;
  rodnecisloNavigation?: Pacient;
}
